src/main/java/net/sf/jabref/model/entry/BibtexEntry.java

```diff
@@ -60,6 +61,14 @@
     private boolean searchHit;
     private boolean groupHit;
 
+    private String serialization;
+
+    /*
+    * marks if the complete serialization, which was read from file, should be used.
+    * Is set to false, if parts of the entry change
+     */
+    private boolean useCustomSerialization;
```

Comment:
I would rename this (and the corresponding methods, of course) as it is always used as "if not changed then..." to "useParsedSerialization" or even to "hasChanged".